### DISCUSSION TITLE
Subscribing to particular types of releases (e.g. based on semver)

### DISCUSSION THREAD

**Discussion by nebbles** [community/community#10073] [Other Feature Feedback, Questions, & Ideas]
I recently started trying out the ability to subscribe specifically to the releases of a repository. Hadn't quite anticipated the activity on the repository!

It would be great if, for subscribing to notifications, I could specify the type of the release. For example, subscribing only to public releases, not pre-releases like alpha/beta/RCs. Even better is if it was possible to subscribe to major/minor/patch of a release.

I for example would most commonly want to subscribe to only major releases or major+minor releases, for a given repository.

**Comment by dchmelik** [comment]
I would like this also.  I watch maybe 100+ projects (and will watch more in future) but mostly only want to know about long-term/stable/user (normally-numbered) not current/development/testing (hexadecimal hash commit-numbered) releases.  I had to redirect all GitHub mail to a separate mailbox and just end up deleting most of it without reading.  Not allowing to watch certain types only causes a pretty big waste of time.
